### PR TITLE
Bugfix/escape backticks in pr body

### PR DESCRIPTION
**Pull request by katybaulch**

### CODE REVIEW

File: auto-tag.sh [hunk 1]
```diff
@@ -13,12 +13,13 @@ fi
 echo "Latest tag: ${latest_tag}"
 
 pr_body="$1"
+pr_tag_section=$("${pr_body}" | awk '/^## Proposed version/,/^### Notes/ {if (!/^## Proposed version/ && !/^Notes:/) print}')
```
Comment (diversemix) [inline auto-tag.sh:16]: If we made this a function in `funcs` then we could add some tests especially for edge cases ... maybe something for later?

File: action.yml [hunk 2]
```diff
@@ -74,7 +74,11 @@ runs:
       id: determine_next_tag_retrospectively
       run: |
         echo "Determining whether user selected an auto-tag option..."
-        script_output=$("${{ github.action_path }}/auto-tag.sh" "${{ fromJson(steps.get_pr_data.outputs.result).body }}" "${{ fromJson(steps.get_pr_data.outputs.result).number }}")
+
+        export PR_BODY=${{ fromJson(steps.get_pr_data.outputs.result).body }}
+        export ESCAPED_PR_BODY="$(echo $PR_BODY | envsubst)"
```
Comment (diversemix) [inline action.yml:79]: Am I missing something - I can't see any env vars in the PR Body?

**Review by diversemix — APPROVED** [review]
Happy to see if this works and we can update / educate me later :smile:

File: auto-tag.sh (hunk 1 above)
Comment (katybaulch) [reply auto-tag.sh:16]: Definitely!

File: action.yml (hunk 2 above)
Comment (katybaulch) [reply action.yml:79]: there aren't... let's see if removing the double quotes here works though, as this is pretty urgent!
